test/test_prototype_utils.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from torchvision.prototype.utils._internal import sequence_to_str
+
+
+@pytest.mark.parametrize(
+    ("seq", "separate_last", "expected"),
+    [
+        pytest.param([], "", "", id="empty"),
+        pytest.param(["foo"], "", "'foo'", id="single"),
+        pytest.param(["foo", "bar"], "", "'foo', 'bar'", id="double"),
+        pytest.param(["foo", "bar"], "and ", "'foo' and 'bar'", id="double-separate_last"),
+        pytest.param(["foo", "bar", "baz"], "", "'foo', 'bar', 'baz'", id="multi"),
+        pytest.param(["foo", "bar", "baz"], "and ", "'foo', 'bar', and 'baz'", id="multi-separate_last"),
```

Comment:
Unless we have a strong reason for keeping them I would prefer avoiding the use of `pytest.param`. They add a lot of extra characters, and make the parametrization harder to read.

For such tests that are super fast to run, do we really need the specify the `id`s?